cli/cmd/plugin/ui/web/tanzu-ui/src/components/VSphereAuthForm.tsx

```diff
@@ -29,8 +31,18 @@ function VSphereAuthForm (props: ChildProps | any) {
 
     const { currentStep, setTabStatus, tabStatus } = props;
     useTabStatus(errors, { currentStep, setTabStatus, tabStatus });
+
     return (
         <CdsFormGroup layout="vertical-inline" control-width="shrink">
+            <CdsToggle>
+                <label>Toggle validation</label>
+                <input type="checkbox" checked={enableValidation} onChange={(e)=> {
+                    if (enableValidation) {
```

Comment:
Does setting enableValidation true 'disable' validation?

Reply:
when the toggle button is clicked, it seems that the onchange callback function is called before enableValidation is changed.

Reply:
I changed the logic a little bit. it should be more readable.